css/style.css

```diff
@@ -80,86 +83,6 @@ body {
     background-size: contain;
 }
 
-.logo-aphrodite {
```

Comment:
IMHO it's much easier if we just put the logo path in the `data.js` instead of building the logo's CSS class based on the slug.